docs/podman-run.1.md

```diff
@@ -653,6 +653,21 @@ Set the UTS mode for the container
 
 **NOTE**: the host mode gives the container access to changing the host's hostname and is therefore considered insecure.
 
+**−−mount**[=*type=<bind|tmpfs>,<src|source>=HOST−DIR,<target|dst,destination>=CONTAINER−DIR[:OPTIONS]*]
+The --mount flag allows you to mount host-directories and tmpfs mounts in a container.
+The --mount flag supports most options that are supported by the -v|--volume flag, but uses a different syntax.
+
+The type must be bind or tmpfs .tmpfs mounted folder data will be lost after container stops.
```

Comment:
Suggest:  The type must be bind or tmpfs.  Note: the .tmpfs mounted directory will not be retained after the container stops.